### PR TITLE
Fix caching success state

### PR DESCRIPTION
**Pull request by pape-io**
- [x] Verified that the failing request works with this fix: http://www.local2.ch:3000/fr/api/weather/locations?areaIds%5B%5D=8001&areaIds%5B%5D=4001&areaIds%5B%5D=3004&areaIds%5B%5D=1000&areaIds%5B%5D=7000&areaIds%5B%5D=6900&areaIds%5B%5D=1950&areaIds%5B%5D=6003

### CODE REVIEW

**Comment by kot-begemot** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ gemspec
 # Declare any dependencies that are still in development here instead of in
 # your gemspec. These might include edge Rails or gems from your path or
 # Git. Remember to move these dependencies to your gemspec before releasing
-# your gem to rubygems.org.
+# your gem to rubygems.org.
```
Comment (Averethel) [inline Gemfile]: No newline at end of file

File: lib/lhc-core-interceptors/caching.rb [hunk 2]
```diff
@@ -44,7 +46,7 @@ def key(request)
       key = "#{request.method.upcase} #{request.url}"
       key += "?#{request.params.to_query}" unless request.params.blank?
     end
-    "LHC_CACHE: #{key}"
+    "LHC_CACHE(#{LHCCoreInterceptors::VERSION}): #{key}"
```
Comment (Averethel) [inline lib/lhc-core-interceptors/caching.rb]: That will invalidate all cache entries on version update, right?

Can we do it only on a need basis? Like having `cache_version` configuration variable maybe?

**Comment by Averethel** [comment]
Small comment & a suggetsion otherwise LGTM!

File: lib/lhc-core-interceptors/caching.rb (hunk 2 above)
Comment (pape-io) [inline lib/lhc-core-interceptors/caching.rb]: Valid point. Using CACHE_VERSIOn for now.

**Pull request merged**